installing/installing_alibaba/installing-alibaba-assisted-installer.adoc

```diff
@@ -0,0 +1,31 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="installing-cluster-alibaba"]
+= Installing on Alibaba Cloud
+include::_attributes/common-attributes.adoc[]
+:context: install-cluster-alibaba
+
+toc::[]
+
+From {product-title} 4.16 and later, you can use the {ai-full} to install an {product-title} cluster on Alibaba Cloud. Alibaba Cloud provides a broad range of cloud computing and data storage services to online businesses and  global enterprises.
+
+[IMPORTANT]
+====
+The {alibaba} installation with {ai-full} is a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
```

Comment:
I recommend that you include the technology-preview snippet and set the attribute instead of hard-coding the snippet here.